src/java.base/share/classes/java/util/Properties.java

```diff
@@ -185,6 +185,7 @@ public Properties() {
      *         accommodate this many elements
      * @throws IllegalArgumentException if the initial capacity is less than
      *         zero.
+     * @since 10
      */
     public Properties(int initialCapacity) {
```

Comment:
This constructor was introduced in Java 10 through https://bugs.openjdk.org/browse/JDK-8189319, so adding `@since 10` here looks correct to me.